### PR TITLE
Add REM usage document into Psammead

### LINKED ISSUE #292: Add rem usage explanation to GEL Foundations Readme
**Is your feature request related to a problem? Please describe.**
Currently, our in-depth explanation about why we use `rem` instead of `em` is in this document in Simorgh https://github.com/bbc/simorgh/blob/67decf15656f785ae20caf0cef7e9e5657920766/docs/Spacing-Units.md 

We should add an explanation within this `gel-foundations` readme. 

**Describe the solution you'd like**
An update to the readme with a concise explanation of `rem` usage here. 

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: N/A

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves #292

**Overall change:**
Add `REM vs EM` document into Psammead.

**Code changes:**
- Add `Spacing-Units.md` markdown file within the `Psammead/documentation` directory 
- Link `gel-foundations/README` `rem` explanation to the document.

---
- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by pjlee11 — APPROVED** [review]

File: documentation/Spacing-Units.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# REMs vs EMs for spacing
+REMs and EMs can be used for spacing (padding/margins/line height). We chose REMs on the basis of clarity and no perceptible difference in rendering.
+
+## REMs
+The REM unit represents a size relative to the root element (e.g., the `font-size` of `<html>`). Combined with the root element's font-size, a REM represents the root element's initial value.
```
Comment (ghost) [inline documentation/Spacing-Units.md]: I don't understand this sentence `Combine with the root element's font-size`. What combined with the root element's `font-size`.

File: documentation/Spacing-Units.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+![rems-win7-ie8](https://user-images.githubusercontent.com/916416/43953553-46899366-9c91-11e8-8335-a0c833a4c5c9.PNG)
+
+
+<hr/>
+
+## EMs
+"Represents the calculated font-size of the element. If used on the font-size property itself, it represents the inherited font-size of the element."(MDN)
+
+### Advantages
+EMs allow for more modularity of components, because spacing will be relative to local font-size.
```
Comment (ghost) [inline documentation/Spacing-Units.md]: Are we taking about `Ems` or `REMs`

File: packages/utilities/gel-foundations/CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 3.1.0 | [PR#701](https://github.com/bbc/psammead/pull/701) Update README `rem` doc link |
```
Comment (ghost) [inline packages/utilities/gel-foundations/CHANGELOG.md]: Should probably be a `3.0.1` as we are not adding any backward compatible functionality to psammead. Its only documentation.

**Review by ghost — CHANGES_REQUESTED** [review]

File: documentation/Spacing-Units.md (hunk 1 above)
Comment (DenisHdz) [reply documentation/Spacing-Units.md]: This is the current content in the [Simorgh doc](https://github.com/bbc/simorgh/blob/latest/docs/Spacing-Units.md), just copied.

Comment (ghost) [reply documentation/Spacing-Units.md]: Yeah but I don't understand what is being combined with the root element's font-size. IMO it doesn't add anything to the sentence. I would remove it.

**Review by ghost — APPROVED** [review]

File: documentation/Spacing-Units.md (hunk 1 above)
Comment (DenisHdz) [reply documentation/Spacing-Units.md]: Replaced :)

**Comment by david-boydell** [comment]
Looks good, this is RFM

**Pull request merged**